checklist.html

```diff
@@ -1,242 +1,1029 @@
+        </p>
+      </div>
+    </details>
+    <details id="section-forms-07">
+      <summary>
+        Update the page or view's <code>title</code> to include high-level form feedback.
+      </summary>
+      <div class="checklist-details">
+        <p class="checklist-criterion">
+          <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html">1.3.1 Info and Relationships</a>
+        </p>
+        <p class="checklist-description">
+          The <code>title</code> element's content is often the first content announced by a screen reader. Adding feedback about the presence of things like errors or warnings after form submission allows people to know there are items on the page or view that need their attention.
+        </p>
+      </div>
+    </details>
+  </div>
+</section>
+
+<section class="article-section resources-section checklist-section" id="section-media" aria-labelledby="label-media">
+  <h2 id="label-media">Media</h2>
+  <p>
+    Media includes content such as pre-recorded and live audio and video.
+  </p>
+  <div class="checklist">
+    <details id="section-media-01">
+      <summary>
+        Make sure that media does not autoplay.
+      </summary>
+      <div class="checklist-details">
+        <p class="checklist-criterion">
+          <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-dis-audio.html">1.4.2 Audio Control</a>
+        </p>
+        <p class="checklist-description">
+          Unexpected video and audio can be distracting and disruptive, especially for certain kinds of cognitive disability such as ADHD. Certain kinds of autoplaying video and animation can be a trigger for vestibular and seizure disorders.
+        </p>
+      </div>
+    </details>
+    <details id="section-media-02">
+      <summary>
+        Ensure that media controls use appropriate markup.
+      </summary>
+      <div class="checklist-details">
+        <p class="checklist-criterion">
+          <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html">1.3.1 Info and Relationships</a>
+        </p>
+        <p class="checklist-description">
+          Examples include making sure a video play button has a pressed toggle state when active, or that a volume slider uses <code>&#x3C;input type=&#x22;range&#x22;&#x3E;</code>.
+        </p>
+      </div>
+    </details>
+    <details id="section-media-03">
+      <summary>
+        Check to see that all media can be paused.
+      </summary>
+      <div class="checklist-details">
+        <p class="checklist-criterion">
+          <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html">2.1.1 Keyboard</a>
+        </p>
+        <p class="checklist-description">
+          Provide a global pause function on any media control. If the device has a keyboard, ensure that pressing the <kbd>Space</kbd> key can pause playback.
```

Comment:
>Provide a global pause function on any media element that can be played.

we need to be careful with:
>ensure that pressing the <kbd>Space</kbd> key can pause playback.

Since space key should allow for the viewport to scroll, when not focusing a form control of some type, we don't want to make people think it's always ok to overwrite this functionality and disallow its inherent behavior.

typically the `space` key should allow for a "global pause" only when keyboard focus is within the context of a larger media grouping, and where space shouldn't also be expected to activate other interactive elements which are focused.

Reply:
Would a followup sentence along the lines of, "Make sure you also don't interfere with the <kbd>Space</kbd> key's ability to scroll the page/view when not focusing on a form control." work for you?

Reply:
mmm.  maybe.  I think using the term "global" is still misleading though.  The <kbd>Space</kbd> key should really only work in the context of the media group.

Reply:
Let's remove the reference to a global function, then:

>Provide the ability to pause any media control. If the device has a keyboard, ensure that pressing the <kbd>Space</kbd> key can pause playback. Make sure you also don't interfere with the <kbd>Space</kbd> key's ability to scroll the page/view when not focusing on a form control.

Sound good?

Reply:
a lot better.  The only thing that is stands out to me now is calling it a "media control".   Swap out "control" with "element" and i think this is good to go.